bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/Connection.java

```diff
@@ -1243,73 +1286,223 @@ public void sendNotificationToMobileApp(String customerId, String text, @Nullabl
+            textToSpeeches.put(json, jsonTextToSpeech);
+        } else {
+            jsonTextToSpeech = textToSpeeches.get(json);
+        }
+        jsonTextToSpeech.devices.add(device);
+        jsonTextToSpeech.ttsVolumes.add(ttsVolume);
+        jsonTextToSpeech.standardVolumes.add(standardVolume);
+        textToSpeechTimer = scheduler.schedule(() -> {
+            sendTextToSpeech();
+        }, 1, TimeUnit.SECONDS);
     }
 
-    private void executeSequenceCommandWithVolume(@Nullable Device device, String command,
-            @Nullable Map<String, Object> parameters, @Nullable Integer ttsVolume, int standardVolume)
-            throws IOException, URISyntaxException {
-        if (ttsVolume != null) {
-            JsonArray nodesToExecute = new JsonArray();
-            Map<String, Object> volumeParameters = new HashMap<>();
-            // add tts volume
-            volumeParameters.clear();
-            volumeParameters.put("value", ttsVolume);
-            nodesToExecute.add(createExecutionNode(device, "Alexa.DeviceControls.Volume", volumeParameters));
-            // add command
-            nodesToExecute.add(createExecutionNode(device, command, parameters));
-            // add volume
-            volumeParameters.clear();
-            volumeParameters.put("value", standardVolume);
-            nodesToExecute.add(createExecutionNode(device, "Alexa.DeviceControls.Volume", volumeParameters));
-
-            executeSequenceNodes(nodesToExecute);
+    private void sendTextToSpeech() {
+        if (textToSpeechTimer != null) {
+            textToSpeechTimer.cancel(true);
+        }
+        for (String json : textToSpeeches.keySet()) {
+            JsonTextToSpeech jsonTextToSpeech = textToSpeeches.get(json);
+            textToSpeechQueue.add(jsonTextToSpeech);
+            if (textToSpeechQueueRunning.compareAndSet(false, true)) {
+                queuedTextToSpeech();
+            }
+        }
+        textToSpeeches.clear();
+    }
+
+    public void queuedTextToSpeech() {
+        JsonTextToSpeech jsonTextToSpeech = textToSpeechQueue.poll();
+        if (jsonTextToSpeech != null) {
+            String text = jsonTextToSpeech.text;
+            try {
+                List<Device> devices = jsonTextToSpeech.devices;
+                List<Integer> ttsVolumes = jsonTextToSpeech.ttsVolumes;
+                List<Integer> standardVolumes = jsonTextToSpeech.standardVolumes;
+
+                Map<String, Object> parameters = new HashMap<>();
+                parameters.put("textToSpeak", text);
+                executeSequenceCommandWithVolume(devices.toArray(new Device[devices.size()]), "Alexa.Speak", parameters,
+                        ttsVolumes.toArray(new Integer[ttsVolumes.size()]),
+                        standardVolumes.toArray(new Integer[standardVolumes.size()]));
+            } catch (IOException | URISyntaxException e) {
+                logger.error("send textToSpeech fails with unexpected error", e);
```

Comment:
reduce to warn, error is reserved for fatal framework failures

Reply:
Ok, i wil change it to "warn".